.github/workflows/lint_and_test.yml

```diff
@@ -55,7 +55,9 @@ jobs:
         run: |
           pytest
       - name: Can run Mentat
-        # Ensure that python doesn't import local mentat folder and that 'mentat' command calls mentat instead of switching folder
+        # Unfortunately Github Actions Runners have trouble with prompt toolkit, so we can't do this on Windows.
```

Comment:
What is the issue exactly for windows?

Reply:
It throws some NoConsoleScreenBufferError exception; it's weird because prompt toolkit works on my windows, so it really is just GHA Windows runners. It throws the error on the creation of a prompt toolkit session.

Reply:
Interesting. I'll make a GH issue for this